stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerDefaultCenterContent.kt

```diff
@@ -65,5 +104,196 @@ internal class MessageComposerDefaultCenterContent : FrameLayout, MessageCompose
                 setText(state.inputValue)
             }
         }
+
+        attachmentsAdapter.setAttachments(state.attachments)
+
+        binding.selectedAttachmentsContainer.isVisible = state.attachments.isNotEmpty()
+        renderValidationErrors(state.validationErrors)
+    }
+
+    /**
+     * Displays first of the validation errors received using in Toast
+     */
+    private fun renderValidationErrors(validationErrors: List<ValidationError>) {
+        if (validationErrors.isEmpty()) return
+
+        val errorMessage = when (val validationError = validationErrors.first()) {
+            is ValidationError.MessageLengthExceeded -> {
+                context.getString(
+                    R.string.stream_ui_message_composer_error_message_length,
+                    validationError.maxMessageLength
+                )
+            }
+            is ValidationError.AttachmentCountExceeded -> {
+                context.getString(
+                    R.string.stream_ui_message_composer_error_attachment_count,
+                    validationError.maxAttachmentCount
+                )
+            }
+            is ValidationError.AttachmentSizeExceeded -> {
+                context.getString(
+                    R.string.stream_ui_message_composer_error_file_size,
+                    MediaStringUtil.convertFileSizeByteCount(validationError.maxAttachmentSize)
+                )
+            }
+        }
+        Toast.makeText(context, errorMessage, LENGTH_SHORT).show()
```

Comment:
For now I've implemented it as in compose. It would be good to improve it later after discussing with designers on both sdks